jieba_test.go

```diff
@@ -12,6 +12,14 @@ func ExampleJieba() {
 	var words []string
 	use_hmm := true
 	//equals with x := NewJieba(DICT_PATH, HMM_PATH, USER_DICT_PATH)
+
+	//equals with x := NewJieba([
+	// 	[DictPath(dict_path string)],
```

Comment:
为什么需要这些DictPath函数呢？我还是有些不太懂。为什么不直接把你自己的词库的路径写进入即可呢？

Reply:
可以不用考虑传参位置。
就可以实现比如 ，我只想修改user_dict_path，我就可以这样调用 
```suggestion
	with x := NewJieba(UserDictPath('user_path'))
```

Reply:
实现类似 Python的关键字参数